test/simplestorage.js

```diff
@@ -1,17 +1,17 @@
-const SimpleStorage = artifacts.require("./SimpleStorage.sol");
+/* eslint-env mocha */
+/* global artifacts contract assert */
```

Comment:
Let's take these two lines out because I don't want to enforce linting on the end user.